packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/plugins/ui/components.tsx

```diff
@@ -172,44 +174,47 @@ export const components = () => {
         </span>
       )
     },
-    [ELEMENT_UL]: ({ attributes, editor, className, element, ...props }) => (
-      <ul
-        className={classNames(
-          blockClasses,
-          className,
-          'mb-4 pl-4 list-disc list-inside last:mb-0'
-        )}
-        {...attributes}
-        {...props}
-      />
-    ),
-    [ELEMENT_OL]: ({ attributes, editor, className, element, ...props }) => (
-      <ol
-        className={classNames(
-          blockClasses,
-          className,
-          'mb-4 pl-4 list-decimal list-inside last:mb-0'
-        )}
-        {...attributes}
-        {...props}
-      />
-    ),
-    [ELEMENT_LI]: ({ attributes, className, editor, element, ...props }) => (
-      <li
-        className={classNames('p-0 mt-0 mb-0 list-outside', className)}
-        {...attributes}
-        {...props}
-      />
-    ),
-    /** "list item content" */
-    [ELEMENT_LIC]: ({ attributes, editor, element, className, ...props }) => (
-      <span
-        // without a min-width the cursor is hidden when the list is empty
-        className={classNames(className, 'w-full inline-block align-top mb-2')}
-        {...attributes}
-        {...props}
-      />
-    ),
+    // [ELEMENT_UL]: ({ attributes, editor, className, element, ...props }) => (
+    // 	<ul
+    // 		className={classNames(
+    // 			blockClasses,
+    // 			className,
+    // 			"mb-4 pl-4 list-disc list-inside last:mb-0",
+    // 		)}
+    // 		{...attributes}
+    // 		{...props}
+    // 	/>
+    // ),
+    // [ELEMENT_OL]: ({ attributes, editor, className, element, ...props }) => (
+    // 	<ol
+    // 		className={classNames(
+    // 			blockClasses,
+    // 			className,
+    // 			"mb-4 pl-4 list-decimal list-inside last:mb-0",
+    // 		)}
+    // 		{...attributes}
+    // 		{...props}
+    // 	/>
+    // ),
+    // [ELEMENT_LI]: ({ attributes, className, editor, element, ...props }) => (
+    // 	<li
+    // 		className={classNames("p-0 mt-0 mb-0 list-outside", className)}
+    // 		{...attributes}
+    // 		{...props}
+    // 	/>
+    // ),
+    // /** "list item content" */
+    // [ELEMENT_LIC]: ({ attributes, editor, element, className, ...props }) => (
+    // 	<span
+    // 		// without a min-width the cursor is hidden when the list is empty
+    // 		className={classNames(className, "w-full inline-block align-top mb-2")}
+    // 		{...attributes}
+    // 		{...props}
+    // 	/>
+    // ),
```

Comment:
**question**: Is this commented-out code worth retaining?

Reply:
Removed